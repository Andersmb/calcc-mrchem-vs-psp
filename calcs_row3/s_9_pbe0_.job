#!/bin/bash

#SBATCH --account=nn4654k
#SBATCH --mail-type=None
#SBATCH --job-name=s_9_pbe0_
#SBATCH --output=s_9_pbe0_.log
#SBATCH --error=s_9_pbe0_.err
#SBATCH --time=00-48:00:00
#SBATCH --mem=160GB
#SBATCH --ntasks=30
#SBATCH --cpus-per-task=6

module purge
module load foss/2021b
module load OpenMPI/4.0.3-GCC-9.3.0
module load Python/3.8.2-GCCcore-9.3.0

cp s_9_pbe0_.inp $SCRATCH
cd $SCRATCH
export OMP_NUM_THREADS=6
$HOME/software/mrchem/install_upstream_master/bin/mrchem --launcher 'srun -n 30'  s_9_pbe0_

savefile s_9_pbe0_.out
savefile s_9_pbe0_.json

DIR=/cluster/projects/nn4654k/$(whoami)/MWOrbitals/${SLURM_JOBID}
mkdir -p $DIR
cp orbitals/* $DIR/
echo $DIR > ${SLURM_SUBMIT_DIR}/s_9_pbe0_.orbitals

DIR=/cluster/projects/nn4654k/$(whoami)/MWCheckpoints/${SLURM_JOBID}
mkdir -p $DIR
cp checkpoint/* $DIR/
echo $DIR > ${SLURM_SUBMIT_DIR}/s_9_pbe0_.checkpoint

exit 0