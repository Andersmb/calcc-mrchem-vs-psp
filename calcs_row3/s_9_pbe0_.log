Starting job 5688411 on c1-[24,36,38],c2-49,c3-18,c5-49 at Fri May 20 18:44:41 CEST 2022

Copying s_9_pbe0_.out to /cluster/home/ambr/projects/psp/row3/calcs/doublets
Copying s_9_pbe0_.json to /cluster/home/ambr/projects/psp/row3/calcs/doublets

Task and CPU usage stats:
JobID           JobName  AllocCPUS   NTasks     MinCPU MinCPUTask     AveCPU    Elapsed ExitCode 
------------ ---------- ---------- -------- ---------- ---------- ---------- ---------- -------- 
5688411       s_9_pbe0_        180                                             02:09:55      0:0 
5688411.bat+      batch         36        1   00:00:04          0   00:00:04   02:09:55      0:0 
5688411.ext+     extern        180        6   00:00:00          0   00:00:00   02:09:55      0:0 
5688411.0      mrchem.x        180       30   02:08:07         29   05:59:21   02:08:23      0:0 

Memory usage stats:
JobID            MaxRSS MaxRSSTask     AveRSS MaxPages   MaxPagesTask   AvePages 
------------ ---------- ---------- ---------- -------- -------------- ---------- 
5688411                                                                          
5688411.bat+     19344K          0     19344K        0              0          0 
5688411.ext+          0          0          0        0              0          0 
5688411.0     34627332K         29 109044603+        0             11          0 

Disk usage stats:
JobID         MaxDiskRead MaxDiskReadTask    AveDiskRead MaxDiskWrite MaxDiskWriteTask   AveDiskWrite 
------------ ------------ --------------- -------------- ------------ ---------------- -------------- 
5688411                                                                                               
5688411.bat+       19.45M               0         19.45M        0.08M                0          0.08M 
5688411.ext+        0.00M               0          0.00M            0                0              0 
5688411.0           6.78M               0          6.74M      255.17M                1        138.52M 

Job 5688411 completed at Fri May 20 20:54:37 CEST 2022
