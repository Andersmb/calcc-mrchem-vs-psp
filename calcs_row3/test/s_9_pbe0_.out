                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               2
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       S            :        0.000000        0.000000        0.384313
    1       F            :        0.000000        0.000000       -2.597864
    2       F            :        2.214146        2.214146        0.487303
    3       F            :       -2.214146        2.214146        0.487303
    4       F            :        2.214146       -2.214146        0.487303
    5       F            :       -2.214146       -2.214146        0.487303
---------------------------------------------------------------------------
 Center of mass          :       -0.000000       -0.000000       -0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                              31
 Beta electrons          :                                              30
 Total electrons         :                                              61
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              9.999998321001e-01
    1    1    a          :                              1.000000000024e+00
    2    1    a          :                              1.000000032643e+00
    3    1    a          :                              1.000000047576e+00
    4    1    a          :                              9.999997691964e-01
    5    1    a          :                              9.999984429763e-01
    6    1    a          :                              9.999998261800e-01
    7    1    a          :                              1.000000007292e+00
    8    1    a          :                              9.999999686854e-01
    9    1    a          :                              1.000000000179e+00
   10    1    a          :                              9.999999196213e-01
   11    1    a          :                              9.999999330328e-01
   12    1    a          :                              9.999999726952e-01
   13    1    a          :                              1.000000222090e+00
   14    1    a          :                              9.999996954763e-01
   15    1    a          :                              9.999994279914e-01
   16    1    a          :                              9.999998700380e-01
   17    1    a          :                              9.999995937407e-01
   18    1    a          :                              9.999992248899e-01
   19    1    a          :                              1.000000967418e+00
   20    1    a          :                              1.000000048180e+00
   21    1    a          :                              9.999999190623e-01
   22    1    a          :                              9.999992907520e-01
   23    1    a          :                              9.999997127652e-01
   24    1    a          :                              1.000001019018e+00
   25    1    a          :                              9.999996525818e-01
   26    1    a          :                              1.000000317142e+00
   27    1    a          :                              1.000000895318e+00
   28    1    a          :                              1.000000332758e+00
   29    1    a          :                              1.000001238346e+00
   30    1    a          :                              1.000000383158e+00
   31    1    b          :                              9.999998264088e-01
   32    1    b          :                              1.000000007049e+00
   33    1    b          :                              9.999999984410e-01
   34    1    b          :                              1.000000054291e+00
   35    1    b          :                              9.999997323220e-01
   36    1    b          :                              9.999984259897e-01
   37    1    b          :                              9.999998259937e-01
   38    1    b          :                              1.000000007292e+00
   39    1    b          :                              9.999999691228e-01
   40    1    b          :                              1.000000000206e+00
   41    1    b          :                              9.999999294377e-01
   42    1    b          :                              1.000000077483e+00
   43    1    b          :                              9.999999931771e-01
   44    1    b          :                              1.000000191993e+00
   45    1    b          :                              9.999996555694e-01
   46    1    b          :                              9.999994279925e-01
   47    1    b          :                              9.999998560946e-01
   48    1    b          :                              9.999995858827e-01
   49    1    b          :                              9.999992253251e-01
   50    1    b          :                              1.000000967454e+00
   51    1    b          :                              1.000000048094e+00
   52    1    b          :                              9.999999192010e-01
   53    1    b          :                              9.999992908611e-01
   54    1    b          :                              9.999997127442e-01
   55    1    b          :                              1.000001019019e+00
   56    1    b          :                              9.999998163021e-01
   57    1    b          :                              1.000000317273e+00
   58    1    b          :                              1.000000895004e+00
   59    1    b          :                              1.000000332800e+00
   60    1    b          :                              1.000001238732e+00
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                887.448843333348
 E-N energy              :            (au)              -2929.351265356772
 Coulomb energy          :            (au)                823.018178747571
 Exchange energy         :            (au)                -18.885654810156
 X-C energy              :            (au)                -58.731731152545
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                404.878529454144
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)              -1296.501629238555
 Nuclear energy          :            (au)                404.878529454144
---------------------------------------------------------------------------
 Total energy            :            (au)             -8.916230997844e+02
                         :      (kcal/mol)             -5.595019378938e+05
                         :        (kJ/mol)             -2.340956109667e+06
                         :            (eV)             -2.426229948771e+04
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -89.265504507201
    1    1    a          :            (au)                -24.602889058240
    2    1    a          :            (au)                -24.585648910695
    3    1    a          :            (au)                -24.579719320371
    4    1    a          :            (au)                -24.534218502356
    5    1    a          :            (au)                -24.530141463215
    6    1    a          :            (au)                 -8.681721381173
    7    1    a          :            (au)                 -6.628954576766
    8    1    a          :            (au)                 -6.627982163858
    9    1    a          :            (au)                 -6.627828524305
   10    1    a          :            (au)                 -1.440699676071
   11    1    a          :            (au)                 -1.286607000853
   12    1    a          :            (au)                 -1.273911674592
   13    1    a          :            (au)                 -1.258109838909
   14    1    a          :            (au)                 -1.206418667975
   15    1    a          :            (au)                 -0.892420026009
   16    1    a          :            (au)                 -0.655260924864
   17    1    a          :            (au)                 -0.649529440501
   18    1    a          :            (au)                 -0.632657802803
   19    1    a          :            (au)                 -0.478760256809
   20    1    a          :            (au)                 -0.470293836962
   21    1    a          :            (au)                 -0.465765687204
   22    1    a          :            (au)                 -0.456285887940
   23    1    a          :            (au)                 -0.445330998588
   24    1    a          :            (au)                 -0.429460711922
   25    1    a          :            (au)                 -0.415909115042
   26    1    a          :            (au)                 -0.410242605144
   27    1    a          :            (au)                 -0.403880954620
   28    1    a          :            (au)                 -0.392225784773
   29    1    a          :            (au)                 -0.375738602259
   30    1    a          :            (au)                 -0.326936296758
   31    1    b          :            (au)                -89.260767456770
   32    1    b          :            (au)                -24.597827263916
   33    1    b          :            (au)                -24.581708938437
   34    1    b          :            (au)                -24.578618877681
   35    1    b          :            (au)                -24.529984986722
   36    1    b          :            (au)                -24.525609339516
   37    1    b          :            (au)                 -8.674777114042
   38    1    b          :            (au)                 -6.624753307036
   39    1    b          :            (au)                 -6.623831452566
   40    1    b          :            (au)                 -6.618647156140
   41    1    b          :            (au)                 -1.429653511941
   42    1    b          :            (au)                 -1.271583592181
   43    1    b          :            (au)                 -1.264039386013
   44    1    b          :            (au)                 -1.244809594895
   45    1    b          :            (au)                 -1.191832551323
   46    1    b          :            (au)                 -0.833614875122
   47    1    b          :            (au)                 -0.636791808181
   48    1    b          :            (au)                 -0.618825088367
   49    1    b          :            (au)                 -0.617056277850
   50    1    b          :            (au)                 -0.467784767634
   51    1    b          :            (au)                 -0.465522969876
   52    1    b          :            (au)                 -0.458612019987
   53    1    b          :            (au)                 -0.444745868206
   54    1    b          :            (au)                 -0.422733262281
   55    1    b          :            (au)                 -0.420963892246
   56    1    b          :            (au)                 -0.411151980808
   57    1    b          :            (au)                 -0.398088201165
   58    1    b          :            (au)                 -0.395656006726
   59    1    b          :            (au)                 -0.385858525082
   60    1    b          :            (au)                 -0.370761881745
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -509.397666153234
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          7.810250e+00        -891.623099784411         -8.916231e+02
    1          5.298356e-01        -896.683438262454         -5.060338e+00
    2          2.061884e-01        -896.906860290969         -2.234220e-01
    3          1.188064e-01        -896.952550028744         -4.568974e-02
    4          6.471132e-02        -896.967722798910         -1.517277e-02
    5          7.547472e-02        -896.981757888416         -1.403509e-02
    6          2.396535e-02        -896.983730410651         -1.972522e-03
    7          2.829952e-02        -896.985278428628         -1.548018e-03
    8          1.174746e-02        -896.985527961544         -2.495329e-04
    9          1.502005e-02        -896.985738208003         -2.102465e-04
   10          6.978069e-03        -896.985828144176         -8.993617e-05
   11          8.836187e-03        -896.985911157449         -8.301327e-05
   12          4.617384e-03        -896.985938423515         -2.726607e-05
   13          4.318542e-03        -896.985956340838         -1.791732e-05
   14          3.946083e-03        -896.985973128042         -1.678720e-05
   15          2.329050e-03        -896.985979294096         -6.166055e-06
   16          1.472681e-03        -896.985981756273         -2.462177e-06
   17          1.059789e-03        -896.985983214911         -1.458638e-06
   18          7.726071e-04        -896.985984156617         -9.417060e-07
   19          5.426214e-04        -896.985984544368         -3.877510e-07
   20          4.154382e-04        -896.985984734104         -1.897363e-07
   21          2.956208e-04        -896.985984849954         -1.158498e-07
   22          1.755071e-04        -896.985984907973         -5.801849e-08
   23          1.485760e-04        -896.985984940864         -3.289097e-08
   24          1.251898e-04        -896.985984959289         -1.842545e-08
---------------------------------------------------------------------------
                      SCF converged in 24 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               2
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       S            :        0.000000        0.000000        0.384313
    1       F            :        0.000000        0.000000       -2.597864
    2       F            :        2.214146        2.214146        0.487303
    3       F            :       -2.214146        2.214146        0.487303
    4       F            :        2.214146       -2.214146        0.487303
    5       F            :       -2.214146       -2.214146        0.487303
---------------------------------------------------------------------------
 Center of mass          :       -0.000000       -0.000000       -0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                894.627249359047
 E-N energy              :            (au)              -2945.404884148248
 Coulomb energy          :            (au)                826.319090631381
 Exchange energy         :            (au)                -18.800279279776
 X-C energy              :            (au)                -58.605690975837
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                404.878529454144
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)              -1301.864514413433
 Nuclear energy          :            (au)                404.878529454144
---------------------------------------------------------------------------
 Total energy            :            (au)             -8.969859849593e+02
                         :      (kcal/mol)             -5.628671991222e+05
                         :        (kJ/mol)             -2.355036362656e+06
                         :            (eV)             -2.440823102118e+04
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -89.382232230369
    1    1    a          :            (au)                -24.866760207549
    2    1    a          :            (au)                -24.832224056892
    3    1    a          :            (au)                -24.832208416915
    4    1    a          :            (au)                -24.832197267512
    5    1    a          :            (au)                -24.832176383131
    6    1    a          :            (au)                 -8.319128077523
    7    1    a          :            (au)                 -6.255702552570
    8    1    a          :            (au)                 -6.255258235445
    9    1    a          :            (au)                 -6.255257892970
   10    1    a          :            (au)                 -1.388327611043
   11    1    a          :            (au)                 -1.296309353567
   12    1    a          :            (au)                 -1.289650340602
   13    1    a          :            (au)                 -1.289626953091
   14    1    a          :            (au)                 -1.255445852650
   15    1    a          :            (au)                 -0.850842160937
   16    1    a          :            (au)                 -0.667844966397
   17    1    a          :            (au)                 -0.667829582834
   18    1    a          :            (au)                 -0.663488257069
   19    1    a          :            (au)                 -0.557614951966
   20    1    a          :            (au)                 -0.557602624516
   21    1    a          :            (au)                 -0.555292803809
   22    1    a          :            (au)                 -0.541828784078
   23    1    a          :            (au)                 -0.514123854810
   24    1    a          :            (au)                 -0.486555423441
   25    1    a          :            (au)                 -0.486127892046
   26    1    a          :            (au)                 -0.486112154532
   27    1    a          :            (au)                 -0.477786940304
   28    1    a          :            (au)                 -0.477772604382
   29    1    a          :            (au)                 -0.450906962849
   30    1    a          :            (au)                 -0.349457396951
   31    1    b          :            (au)                -89.377710478309
   32    1    b          :            (au)                -24.867063091777
   33    1    b          :            (au)                -24.826550468977
   34    1    b          :            (au)                -24.826539139159
   35    1    b          :            (au)                -24.826526193399
   36    1    b          :            (au)                -24.826511064327
   37    1    b          :            (au)                 -8.313247326302
   38    1    b          :            (au)                 -6.252096859889
   39    1    b          :            (au)                 -6.252096052839
   40    1    b          :            (au)                 -6.245286902868
   41    1    b          :            (au)                 -1.382745978890
   42    1    b          :            (au)                 -1.288303389554
   43    1    b          :            (au)                 -1.276009175533
   44    1    b          :            (au)                 -1.275996186017
   45    1    b          :            (au)                 -1.241225798327
   46    1    b          :            (au)                 -0.829876862289
   47    1    b          :            (au)                 -0.654466365534
   48    1    b          :            (au)                 -0.654460502918
   49    1    b          :            (au)                 -0.641039928085
   50    1    b          :            (au)                 -0.551396386296
   51    1    b          :            (au)                 -0.546875184039
   52    1    b          :            (au)                 -0.546874531652
   53    1    b          :            (au)                 -0.515657804175
   54    1    b          :            (au)                 -0.507781448409
   55    1    b          :            (au)                 -0.479206313826
   56    1    b          :            (au)                 -0.479198267912
   57    1    b          :            (au)                 -0.473026680180
   58    1    b          :            (au)                 -0.471820028275
   59    1    b          :            (au)                 -0.471815665846
   60    1    b          :            (au)                 -0.445860059243
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -511.320956927597
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       -0.000020       -0.000127       -0.197052
 Magnitude               :            (au)                        0.197052
                         :         (Debye)                        0.500857
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000        0.311126
 Magnitude               :            (au)                        0.311126
                         :         (Debye)                        0.790804
---------------------------------------------------------------------------
 Total vector            :       -0.000020       -0.000127        0.114074
 Magnitude               :            (au)                        0.114074
                         :         (Debye)                        0.289948
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  2h  8m 14s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
